extra-modules/extra-phishing/src/main/kotlin/com/kotlindiscord/kord/extensions/modules/extra/phishing/PhishingExtension.kt

```diff
@@ -331,12 +340,8 @@ class PhishingExtension(private val settings: ExtPhishingBuilder) : Extension()
             }
 
             return url
-        } catch (e: HttpRequestTimeoutException) {
-            logger.warn { "$url -> Request timeout has expired." }
-
-            return url
-        } catch (e: URLParserException) {
-            logger.debug { "$url -> Failed to parse url" }
+        } catch (e: Exception) {
+            logger.debug(e) { e.message }
```

Comment:
Your PR description states that this is a warning, not a debug message. Additionally, you've removed the URL from the log message - is that a good idea?

Furthermore, shouldn't logging the exception object by passing it directly to the logger call also output the exception message automatically?